RectangleClient.java

```diff
@@ -0,0 +1,16 @@
+
+public class RectangleClient {
+	public static void main(String[] args) {
+		
+		Rectangle rect1 = new Rectangle(2,5,8,10);
+		System.out.println(rect1);
+		
+		if (rect1.contains(5,6)) {
```

Comment:
If you're just going to print out "True" and "False", you can just write `System.out.println(rect1.contains(5,6))` - try it!